docs/api.md

```diff
@@ -338,14 +334,14 @@ actually starting the first sync with it.
 The list of imported records is returned.
 
 ```js
-articles.loadDump([
+let records = await articles.loadDump([
   {
     id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
     title: "baz",
     last_modified: 1432222889337
   }
-])
-  .then(records => console.log(records));
+]);
+console.log(records));
```

Comment:
What's the reasoning for which places get `console.log` and which don't?

Reply:
[`console.log`](https://developer.mozilla.org/en-US/docs/Web/API/Console/log) is used whenever you want to log a general value to the console, and [`console.error`](https://developer.mozilla.org/en-US/docs/Web/API/Console/error) is used whenever you want to log an error to the console.
There are also [a few more logging methods](https://developer.mozilla.org/en-US/docs/Web/API/console#Methods) you can check out, notably [`console.log`](https://developer.mozilla.org/en-US/docs/Web/API/Console/log), [`console.error`](https://developer.mozilla.org/en-US/docs/Web/API/Console/error), [`console.warn`](https://developer.mozilla.org/en-US/docs/Web/API/Console/warn) and [`console.info`](https://developer.mozilla.org/en-US/docs/Web/API/Console/info).